diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/FileSizeWarnTest.kt

```diff
@@ -75,4 +85,40 @@ class FileSizeWarnTest {
                         "${Warnings.FILE_IS_TOO_LONG.warnText()} $size", false),
                 rulesConfigList = rulesConfigListLarge)
     }
+
+    @Test
+    fun `configuration is empty`(){
+        val path = javaClass.classLoader.getResource("test/paragraph3/src/main/FileSizeLarger.kt")
+        val file = File(path!!.file)
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListEmpty)
+    }
+
+    @Test
+    fun `file has more than 2000 lines`(){
+        val path = javaClass.classLoader.getResource("test/paragraph3/src/main/A/FileSize2000.kt")
+        val file = File(path!!.file)
+        val size = generate2000lines() + 2
+        lintMethod(FileSize(), file.absolutePath, file.readText(),
+                LintError(1, 1, "$DIKTAT_RULE_SET_ID:file-size",
+                        "${Warnings.FILE_IS_TOO_LONG.warnText()} $size", false),
+                rulesConfigList = rulesConfigListLarge)
+    }
+
+    @Test
+    fun `config has only ignoreFolders`(){
+        val path = javaClass.classLoader.getResource("test/paragraph3/src/main/A/FileSize2000.kt")
+        val file = File(path!!.file)
+        val size = generate2000lines() + 2
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListOnlyIgnore)
+    }
+
+    private fun generate2000lines(): Int{
+        val path = javaClass.classLoader.getResource("test/paragraph3/src/main/A/FileSize2000.kt")
+        val file = File(path!!.file)
+        val text = StringBuilder()
+        for (i in 0..2000)
+            text.append("//hello \n")
```

Comment:
In Kotlin you can use `String`'s method `repeat` without the need to explicitly use `StringBuilder`. You can have a look in kotlin sources, it actually uses `StringBuilder` under the hood.